src/index.html

```diff
@@ -7,8 +7,74 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin="use-credentials" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="logo-MOYO">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
```

Comment:
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
```
don't add space between parent and child element